dns-records.tf

```diff
@@ -196,6 +201,43 @@ resource "azurerm_dns_cname_record" "jenkinsio_target_public_prodpublick8s" {
   })
 }
 
+# jenkins.io CNAME records pointing to status incident for maintenance reasons
+resource "azurerm_dns_cname_record" "jenkinsio_target_status_incident" {
+  # Map of records and corresponding purposes
+  for_each = {
+    "accounts"      = "accountapp for Jenkins users, redirected to status.jenkins.io while migrating LDAP, see https://github.com/jenkins-infra/helpdesk/issues/3351#issuecomment-1576741071"
+  }
+
+  name                = each.key
+  zone_name           = data.azurerm_dns_zone.jenkinsio.name
+  resource_group_name = data.azurerm_resource_group.proddns_jenkinsio.name
+  ttl                 = 60
+  record              = "status.jenkins.io/issues/2023-06-08-ldap-migration-to-publick8s"
+
+  tags = merge(local.default_tags, {
+    purpose = each.value
+  })
+}
+
+# jenkins-ci.org CNAME records pointing to status incident for maintenance reasons
+resource "azurerm_dns_cname_record" "jenkinsciorg_target_status_incident" {
+  # Map of records and corresponding purposes. Some records only exists in jenkins.io as jenkins-ci.org is only legacy
+  for_each = {
+    "accounts" = "accountapp for Jenkins users, redirected to status.jenkins.io while migrating LDAP, see https://github.com/jenkins-infra/helpdesk/issues/3351#issuecomment-1576741071"
+  }
+
+  name                = each.key
+  zone_name           = data.azurerm_dns_zone.jenkinsciorg.name
+  resource_group_name = data.azurerm_resource_group.proddns_jenkinsci.name
+  ttl                 = 60
+  record              = "status.jenkins.io/issues/2023-06-08-ldap-migration-to-publick8s"
```

Comment:
```suggestion
  record              = "status.jenkins.io"
```

It is only DNS: there is no concept of URL or URI here. Your CNAME can only point to another domain name.